src/views/parents/parents.jsx

```diff
@@ -0,0 +1,119 @@
+var React = require('react');
+var FormattedHTMLMessage = require('react-intl').FormattedHTMLMessage;
+var FormattedMessage = require('react-intl').FormattedMessage;
+var render = require('../../lib/render.jsx');
+
+var Page = require('../../components/page/www/page.jsx');
+
+require('./parents.scss');
+
+var Parents = React.createClass({
+    type: 'Parents',
+    render: function () {
+        return (
+            <div className="inner parents">
+                <h1><FormattedMessage id='general.forParents' /></h1>
+
+                <div className="masthead">
+                    <div>
+                        <p><FormattedMessage id='parents.introOne' /></p>
+                        <p><FormattedMessage id='parents.introTwo' /></p>
+
+                        <ul>
+                            <li><a href = "/about/"><FormattedMessage id="parents.introAbout" /></a></li>
+                            <li><a href = "/educators/"><FormattedMessage id="parents.introEducators" /></a></li>
+                        </ul>
+                    </div>
+
+                    <div>
+                        <iframe
+                            title="Mitch TED Talk"
+                            src="//embed-ssl.ted.com/talks/mitch_resnick_let_s_teach_kids_to_code.html"
+                            frameBorder="0"
+                            webkitAllowFullScreen
+                            mozallowfullscreen
+                            allowFullScreen />
+                    </div>
+                </div>
+
+                <div className="body">
+                    <ul>
+                      <li>
+                            <h3><FormattedMessage id='parents.learning' /></h3>
```

Comment:
The "Learning", "Questions" and "community" sections on the old page are on boxes that are on the right side of the screen – I think that, for now, it makes sense to try and get those on the right side for consistency. I think it might be do-able by using [`FlexRow`](https://github.com/LLK/scratch-www/blob/develop/src/components/flex-row/flex-row.scss#L15), perhaps by having the questions in one `ul` and the "Learning", "Questions" and "Community" in another. Not sure though.